recipes-debian/python/python-native_debian.bb

```diff
@@ -0,0 +1,120 @@
+# base recipe: meta/recipes-devtools/python/python-native_2.7.16.bb
+# base branch: warrior
+# base commit: 2268bf548a2f7e06da9071b11b4c7a487859778d
+
+require python.inc
+EXTRANATIVEPATH += "bzip2-native"
+DEPENDS = "openssl-native bzip2-replacement-native zlib-native readline-native sqlite3-native expat-native gdbm-native db-native"
+
+inherit debian-package
+require recipes-debian/sources/python2.7.inc
+
+LICENSE = "PSFv2"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=e466242989bd33c1bd2b6a526a742498"
+
+DEBIAN_UNPACK_DIR = "${WORKDIR}/Python-${PV}"
+BPN = "python2.7"
+DEBIAN_PATCH_TYPE = "quilt"
+
+FILESPATH_append = ":${COREBASE}/meta/recipes-devtools/python/python-native"
+FILESPATH_append = ":${COREBASE}/meta/recipes-devtools/python/python"
+FILESPATH_append = ":${THISDIR}/python"
+
+SRC_URI += "\
+            file://05-enable-ctypes-cross-build.patch \
+            file://10-distutils-fix-swig-parameter.patch \
+            file://11-distutils-never-modify-shebang-line.patch \
+            file://debug.patch \
+            file://unixccompiler.patch \
+            file://multilib-debian.patch \
+            file://add-md5module-support.patch \
+            file://builddir.patch \
+            file://revert_use_of_sysconfigdata.patch \
+            file://0001-python-native-fix-one-do_populate_sysroot-warning.patch \
+            file://bpo-35907-cve-2019-9948-fix.patch \
+            file://bpo-36216-cve-2019-9636-fix.patch \
+            file://dont-use-multiarch.patch \
+           "
+
+FILESEXTRAPATHS =. "${FILE_DIRNAME}/${PN}:"
+
+inherit native
+
+EXTRA_OECONF_append = " --bindir=${bindir}/${PN} --with-system-expat=${STAGING_DIR_HOST}"
+
+EXTRA_OEMAKE = '\
+  LIBC="" \
+  STAGING_LIBDIR=${STAGING_LIBDIR_NATIVE} \
+  STAGING_INCDIR=${STAGING_INCDIR_NATIVE} \
+'
+
+do_debian_patch_prepend() {
+    export DEB_TARGET_ARCH=${DPKG_ARCH}
+    export DEB_TARGET_ARCH_OS=${TARGET_OS}
+
+    cd ${DEBIAN_UNPACK_DIR}
+    chmod +x ./debian/rules
+    # Generage ./debian/patches/series
+    ./debian/rules ./debian/patches/series
+}
+
+do_debian_patch_append() {
```

Comment:
How about remove multiarch-libc.diff from ./debian/patches/series after generate debian/patches/series file in do_debian_patch_prepend() ?